More Enemy AI!

kszalai opened the pull request:
Added Prefabs for these items
Gopher
Pond
Tees

Added Scripts for these items
Gopher
Pond
Tees
SandTrap

grizzlymanbear committed:
Update README

grizzlymanbear committed:
Merge commit '5f9974c367b024e90bc7695a5a78074183cc04d6' into Jared

grizzlymanbear committed:
Merge branch 'master' of https://github.com/kylewild/COSC457_Game into Jared

grizzlymanbear committed:
Created Main Menu

grizzlymanbear committed:
Merge branch 'Single-Player-UI' of https://github.com/kylewild/COSC457_Game into Jared

grizzlymanbear committed:
Added music file to project

grizzlymanbear committed:
Updated font of buttons/label

kszalai committed:
Created more Prefabs and Scripts

Prefabs created
Tees (Plane temporarily)
Pond (Plane)
Gopher (Cube, potentially find a model like we did for the Alligator
later?)

Scripts created
Gopher (Empty for now. Need to detect when player is near, but we have
no player model to use right now)
SandTrap (OnTriggerEnter method created, but empty due to no player
model)
Pond (Same as SandTrap)
Tees (Same as SandTrap)
--
Kyle

kszalai committed:
Updated Gopher Prefab

Made a Capsule instead of a Cube

hounvs committed:
Merge remote-tracking branch 'refs/remotes/origin/master' into Jared

Conflicts:
	Fore Score and Seven Beers Ago/Fore Score and Seven Beers Ago.sln

kszalai committed:
Updated Gopher Prefab and Script

Changed from Cube to Capsule

hounvs commented:
Checking this now. ill merge when done

hounvs committed:
Merge remote-tracking branch 'refs/remotes/origin/master' into Enemy-AI

hounvs committed:
Merge remote-tracking branch 'refs/remotes/origin/Jared' into Enemy-AI

hounvs committed:
Pull in Jared's branch

Pull in Jared's branch, make a few adjustments to work with this branch

hounvs committed:
Update SPLevel1 and obstacle prefabs

Set all obstacle prefabs to start at origin

Add/change obstacles in level

hounvs commented:
Gopher AI isn't working. Everything else is though. Looking into fixing still. Will merge with master when done. As I type this, Kyle says he fixed it. WE SHALL SEEEEEEEEEEEEE

kszalai committed:
Fixed Gopher

Previous method was weird
This one works. Added flag for AboveGround. No longer will try to stay
above ground for 2 seconds, it will be above ground for 5 seconds, and
then hide for 5 seconds.

kszalai commented:
OH MAN, HOW DO MODIFY A PULL REQUEST

hounvs committed:
Update gitignore to include Visual Studio

hounvs committed:
Add GopherHole

Add hole where gopher is so you can see it while it is underground. Will
need to be beautified in the future
